Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -225,7 +225,7 @@ function Add-SandboxCredentials {
         [string] $populatedSecret = (Get-RandomString),
         [string] $minimalKey = (Get-RandomString),
         [string] $minimalSecret = (Get-RandomString),
-        [string] $adminCredentialConfigPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'Application\EdFi.Ods.SandboxAdmin.Web\AdminCredential.config'),
+        [string] $adminCredentialConfigPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'Application\EdFi.Ods.SandboxAdmin\AdminCredential.config'),
```

Comment:
should this be the appsettings.user.json instead?

Reply:
This function is deprecated and can be removed from running as part of initdev. Let's make sure these settings are now correctly coming from the settings-management.psm1

Reply:
I removed the call to this function in initdev.